### PR TITLE
implement connect

### PR DESCRIPTION
**Pull request by aherlihy**

### CODE REVIEW

File: packages/history/src/history.ts [hunk 1]
```diff
@@ -11,12 +12,16 @@ import redactInfo from 'mongodb-redact';
...
 export function changeHistory(history: string[], redact = false): void {
   const hiddenCommands =
-    RegExp('createUser|auth|updateUser|changeUserPassword', 'g');
+    RegExp('createUser|auth|updateUser|changeUserPassword|connect', 'g');
 
   if (hiddenCommands.test(history[0])) {
     history.shift();
     return;
   }
+  if (/Mongo\(([^+)]+)\)/g.test(history[0])) {
```
[the remark this answers is not in the document]
Comment (lrlna) [reply packages/history/src/history.ts:21]: hmm what should the URI look like when using the Mongo object? Can I do this?
```js
new Mongo(mongodb+srv://admin:catscat3ca1s@cats-data-sets-e08dy.mongodb.net/admin)
```

[the remark this answers is not in the document]
Comment (aherlihy) [reply packages/history/src/history.ts:21]: The URI format is pretty universal, I don't think its any different from any other URI instance

[the remark this answers is not in the document]
Comment (aherlihy) [reply packages/history/src/history.ts:21]: Would that not work with the CLI redact credentials helper?

[the remark this answers is not in the document]
Comment (lrlna) [reply packages/history/src/history.ts:21]: ah gotcha, I guess that regex does not work for me with the example I posted. I am not able to extract the URI. It's meant to match on the entire URI string?
Screen Shot 2020-06-11 at 12 00 00

[the remark this answers is not in the document]
Comment (lrlna) [reply packages/history/src/history.ts:21]: I see that you have a test for this, and it does look like it works, so I am surprised why I can't get the URI with srv to work.

[the remark this answers is not in the document]
Comment (aherlihy) [reply packages/history/src/history.ts:21]: You're right, I think it's the extra `+`, I've removed it and now it seems to recognize both a regular username/pwd URI and a mongodb SRV

[the remark this answers is not in the document]
Comment (aherlihy) [reply packages/history/src/history.ts:21]: Nice catch!!

**Pull request merged**